Dockerfile

```diff
@@ -1,20 +1,21 @@
-FROM --platform=linux/amd64 alpine/helm:3.10.2
+FROM --platform=linux/amd64 alpine/helm:3.13.3
+# Helm supported version along with K8 version: https://helm.sh/docs/topics/version_skew/
+
 LABEL maintainer="Yann David (@Typositoire) <davidyann88@gmail>"
 
 # Versions for gcloud, kubectl, doctl, awscli
 # K8 versions: https://kubernetes.io/releases/
-ARG KUBERNETES_VERSION=1.26.0
+ARG KUBERNETES_VERSION=1.27.11
 ARG GCLOUD_VERSION=416.0.0
 ARG DOCTL_VERSION=1.57.0
-# https://pypi.org/project/awscli/
-ARG AWSCLI_VERSION=1.31.10
+ARG AWSCLI_VERSION=2.13.25-r0
 ARG HELM_PLUGINS_TO_INSTALL="https://github.com/databus23/helm-diff"
 
 #gcloud path
 ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
 
 #install packages
-RUN apk add --update --upgrade --no-cache jq bash curl git gettext libintl py-pip
+RUN apk add --update --upgrade --no-cache jq bash curl git gettext libintl py-pip aws-cli=${AWSCLI_VERSION}
```

Comment:
It might be nice to standardize on either using `curl` or `wget` for our dockerfiles, we currently use both.

Reply:
Ahh I hadn't picked up that, it was a also a chore that I've meaning to do for some of our own dockerfiles.

Reply:
Oh you probably don't need to install `pip` then if you're not using it to install `awscli`.

Reply:
`py-pip` was already a dependency

Reply:
I emailed the author (who made me a contributor to the repo), but the CI checks are broken